src/System.Net.Http.WinHttpHandler/tests/UnitTests/ClientCertificateHelper.cs

```diff
@@ -0,0 +1,271 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Security.Cryptography.X509Certificates;
+
+using Xunit;
+
+namespace System.Net.Http.WinHttpHandlerUnitTests
+{
+    public class ClientCertificateHelper
+    {
+        private readonly X509Certificate2 _cert_KeyUsageIncludesDigitalSignature_EKUIncludesClientAuth_PrivateKey =
+            new X509Certificate2(
+                Convert.FromBase64String(
+                  @"MIIKTgIBAzCCCgoGCSqGSIb3DQEHAaCCCfsEggn3MIIJ8zCCBgwGCSqGSIb3DQEHAaCCBf0EggX5
```

Comment:
>Also, no, it wouldn't be Base64 since .Net only works with password protected PFX files for public/private key pairs.

My "PFX" data is password protected with "password" as the password when I created/exported this test data.   .NET works fine with "PFX" byte streams regardless of whether they are stored as byte[] or Base64Encode (and then decode).

I agree that generally speaking we shouldn't mix test code with test data.  That is why I put the  test "data" into a separate helper class.  IMHO it makes these tests easier to maintain and read.